### PR TITLE
Completed MVP DOM-I

### PR DESCRIPTION
**Pull request by marianahuh**
Submitted MVP for DOM-I

### CODE REVIEW

**Comment by marianahuh** [comment]
Changed 'nav' properties for uniformity. @ielvisd

File: js/index.js [hunk 1]
```diff
@@ -14,29 +14,87 @@ const siteContent = {
...
-    "copyright" : "Copyright Great Idea! 2018"
+    "copyright": "Copyright Great Idea! 2018"
   },
 };
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+const anchorElement = document.querySelectorAll("nav a");
```
Comment (ielvisd) [inline js/index.js:44]: Nice use of `querySelectorAll` to target your nav items.

File: js/index.js [hunk 2]
```diff
@@ -14,29 +14,87 @@ const siteContent = {
...
+let h1 = document.querySelector("h1");
+h1.innerHTML = siteContent.cta.h1.replace(new RegExp(" ", 'g'), '<br> ');
+
+const btn = document.querySelector(".cta button");
+btn.textContent = "Get Started";
+
+const ctaElement = document.querySelector("#cta-img");
+ctaElement.setAttribute('src', siteContent["cta"]["img-src"])
+
+const middleImg = document.querySelector("#middle-img");
```
Comment (ielvisd) [inline js/index.js:63]: Good use of a querySelector to change the image source.

File: js/index.js [hunk 3]
```diff
@@ -14,29 +14,87 @@ const siteContent = {
...
+const productText = document.querySelectorAll(".text-content p")[3];
+productText.textContent = siteContent["main-content"]["product-content"]
+
+const visionText = document.querySelectorAll(".text-content p")[4];
+visionText.textContent = siteContent["main-content"]["vision-content"]
+
+const contact = document.querySelector(".contact h4");
+contact.textContent = siteContent["contact"]["contact-h4"]
+
+const contactInfo = document.querySelectorAll(".contact p");
```
Comment (ielvisd) [inline js/index.js:91]: Good use of Query Selector all to update the contact info.

**Review by ielvisd — APPROVED** [review]
Nice job Mariana, Your Great Idea site looks very close to the solution.

**Pull request merged**